cfme/configure/access_control.py

```diff
@@ -168,149 +238,298 @@ def description(self):
 
 @navigator.register(User, 'All')
 class UserAll(CFMENavigateStep):
-    prerequisite = NavigateToAttribute('appliance.server', 'Configuration')
```

Comment:
Why split this like this? To avoid the refresh? This is breaking the model for navmazing because it does two navigations in one go. In the future we are hoping to have some kind of graph of how navigations have taken place, this will show that the AllUser view is accessible from the Logged In page, instead of being from the Configuration page. It also means that if accessing configuration changes, we now need to make that change in another location.